tests/qchem/vibrational/test_vibrational_class.py

```diff
@@ -99,12 +99,12 @@ def test_single_point_energy(sym, geom, unit, method, basis, expected_energy):
         (
             ["H", "F"],
             np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.0]]),
-            np.array([[0.0, 0.0, 0.07497201], [0.0, 0.0, 1.81475336]]),
+            np.array([[0.0, 0.0, 0.03967348], [0.0, 0.0, 0.96032612]]),
         ),
         (
             ["C", "O"],
             np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.0]]),
-            np.array([[0.0, 0.0, -0.12346543], [0.0, 0.0, 2.0131908]]),
+            np.array([[0.0, 0.0, -0.06533509], [0.0, 0.0, 1.06533469]]),
```

Comment:
Changed these as earlier the results were always in Bohr.

Reply:
You need to test both Ang and Bohr, otherwise Codecov will complain.